proc/scielo_doi/app/cisis.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+import os
+from datetime import datetime
+
+
+CISIS = "cisis/mx"
+
+
+def update_command(db, mfn, count, proc):
+    cmd = ("{CISIS} {db} from={mfn} count={count} "
+           "\"proc={proc}\""
+           " copy={db} now -all").format(
+            CISIS=CISIS,
+            db=db,
+            mfn=mfn,
+            count=count,
+            proc=proc
+        )
+    return cmd
+
+
+def execute(cmd):
+    os.system(cmd)
```

Comment:
Sugiro avaliar o uso do módulo `subprocess`, que é de mais alto-nível e oferece facilidades para tratar erros etc.

Reply:
Vou deixar como comentário TODO para uma próxima etapa como um lembrete.

Na verdade, no módulo cisis.py nem deveria haver esta função `execute`, já o módulo tem a intenção de ser importável e quem for usar as funções do módulo poderia usar da forma que quisesse já que retorna o comando a ser executado.